### DISCUSSION TITLE
support for windows 8.1

### DISCUSSION THREAD

**Discussion by jacob755** [Eloston/ungoogled-chromium#2173] [General]
hello
updated UG and see this .so question is UG keep updated for windows 8.1?
i hope so thanks
![screenshot 2022-12-05 001 06 57 43](https://user-images.githubusercontent.com/65178892/205626386-244997fc-3683-47b3-a517-c794a3c7e051.png)

**Comment by PF4Public** [comment]
ungoogled-chromium is a drop-in replacement for Chromium and most of the user experience is preserved from Chromium. Likewise this OS check definitely originates from Chromium itself. And if Chromium codebase stops supporting earlier Windows versions, so is likely ungoogled-chromium.

**Reply by Marek33** [reply]
> ungoogled-chromium is a drop-in replacement for Chromium and most of the user experience is preserved from Chromium. Likewise this OS check definitely originates from Chromium itself. And if Chromium codebase stops supporting earlier Windows versions, so is likely ungoogled-chromium.

That's a pity to hear, since on the Windows 7 it's the same.

However, can we have the **disable-infobar** command returned? I tried it, on the latest **108.0.5359.95-1.1** version, but it doesn't work for me.

**Reply by PF4Public** [reply]
Someone might be enough bothered by this to fix this.

**Comment by tomasz1986** [comment]
Related: https://support.google.com/chrome/thread/185534985/sunsetting-support-for-windows-7-8-8-1-in-early-2023

On a side note, Chromium would install and run on Windows XP and Vista for quite a while even after the official support had ended, so you've probably still got some time left, but the browser is bound to become code-incompatible with the two OSs eventually.
